charactersheet/charactersheet/viewmodels/character/spells.js

```diff
@@ -36,11 +36,14 @@ function SpellbookViewModel() {
     self.sort = ko.observable(self.sorts['spellName asc']);
 
     self.numberOfPrepared = ko.computed(function(){
-        var prepared = ko.utils.arrayFilter(self.spellbook(), function(spell) {
-            return spell.spellPrepared() === true;
+        var prepared = 0;
```

Comment:
I didn't really like the logic. It created an array of items that matched a criteria and then got the length of an array. 
This uses a simple counter and a loop. No observables, arrays, or knockout stuff involved.